tools/validators/instance_validator/validate/point.py

```diff
@@ -25,4 +25,12 @@ class Point(object):
   def __init__(self, point_name, value):
     super().__init__()
     self.point_name = point_name
-    self.present_value = value
+    self.present_value = self._ToString(value)
+
+  def _ToString(self, value):
```

Comment:
I think people (like me) might be confused about why this isn't using the built-in __str__ method. Perhaps calling it _ValueToString will clear up that confusion.